src/components/Menu.vue

```diff
@@ -1,79 +1,19 @@
-            <!--v-for="(n, nid) in nav"-->
-            <!--:key="nid"-->
-            <!--@click="navigateTo(n.route)">-->
-              <!--<v-list-tile-action>-->
-                <!--<v-icon>{{ n.iconMaterial }}</v-icon>-->
-              <!--</v-list-tile-action>-->
-              <!--<v-list-tile-content>-->
-                <!--<v-list-tile-title>{{ n.name }}</v-list-tile-title>-->
-              <!--</v-list-tile-content>-->
-          <!--</v-list-tile>-->
-
-          <!--<v-divider></v-divider>-->
-
-          <!--<v-list-group-->
-                      <!--v-for="item in menuData"-->
-                      <!--v-model="item.active"-->
-                      <!--:key="item.title"-->
-                      <!--no-action-->
-                    <!--&gt;-->
-
-                      <!--<v-list-tile slot="activator">-->
-                        <!--<v-list-tile-action>-->
-                          <!--<v-icon>{{ item.icon }}</v-icon>-->
-                        <!--</v-list-tile-action>-->
-                        <!--<v-list-tile-content>-->
-                          <!--<v-list-tile-title>{{ item.title }}</v-list-tile-title>-->
-                        <!--</v-list-tile-content>-->
-                      <!--</v-list-tile>-->
-                      <!--<v-list-tile-->
-                        <!--v-for="subItem in item.subItems"-->
-                        <!--:key="subItem.title"-->
-                        <!--@click="notImplemented()"-->
-                      <!--&gt;-->
-                        <!--<v-list-tile-content>-->
-                          <!--<v-list-tile-title>{{ subItem.title }}</v-list-tile-title>-->
-                        <!--</v-list-tile-content>-->
-
-                        <!--<v-list-tile-action>-->
-                          <!--<v-icon>{{ subItem.action }}</v-icon>-->
-                        <!--</v-list-tile-action>-->
-                      <!--</v-list-tile>-->
-                    <!--</v-list-group>-->
-
-            <!--<v-divider ></v-divider>-->
-
-            <!--<v-list-tile-->
-              <!--@click="logout()">-->
-              <!--<v-list-tile-action>-->
-                <!--<v-icon>exit_to_app</v-icon>-->
-              <!--</v-list-tile-action>-->
-              <!--<v-list-tile-content>-->
-                <!--<v-list-tile-title><translate :translate-params="{name: configuration.theme.general.name}">Exit %{name}</translate></v-list-tile-title>-->
-              <!--</v-list-tile-content>-->
-            <!--</v-list-tile>-->
-
-        <!--</v-list>-->
-      <!--</v-navigation-drawer>-->
+  <oc-left-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false">
+    <template slot="default">
+      <oc-sidebar-nav-item v-for="(n, nid) in nav" :key="nid" :text="n.name" :icon="n.iconMaterial" :target="n.route" />
+      <oc-sidebar-nav-divider />
+      <oc-sidebar-nav-item text="Exit ownCloud" active icon="exit_to_app" target="login" />
```

Comment:
see L62 for translations

Reply:
https://github.com/owncloud/phoenix/pull/839/files#diff-597f7240e2b769bbd70fb848c323b0d7L62